### PR TITLE
Fixes #326: refactor message section

### LINKED ISSUE #326: Refactor Message Section
**Actual Behaviour**

Currently, the message section is too complicated.

**Expected Behaviour**

It needs to be refactored.

**Would you like to work on the issue?**

Anyone can take this up.

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #326 

Changes: Made Separate DialogSection made for all the dialogs in message section.

Demo Link: http://susimadhav.surge.sh/
@uday96 @rishiraj824 @daminisatya @isuruAb Please Review
Thanks

### CODE REVIEW

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< 72abf853cda4c560ad611fd7183f972c5b47eabb:src/components/ChatApp/MessageSection.react.js
```
Comment (isuruAb) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: conflict here check this again

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 2]
```diff
@@ -558,57 +570,18 @@ class MessageSection extends Component {
...
+              ThemeChangerComponents={components}
               bodyStyle={bodyStyle}
-              onRequestClose={this.handleHardwareToggle.bind(this, false)}>
-              <div>
-                <HardwareComponent {...this.props} />
-              </div>
-            </Dialog>
-            <Dialog
               actions={actions}
+<<<<<<< 72abf853cda4c560ad611fd7183f972c5b47eabb:src/components/ChatApp/MessageSection.react.js
```
Comment (isuruAb) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: what is this line ?

**Comment by mariobehling** [comment]
Please resolve merge conflicts.

**Comment by madhavrathi** [comment]
@mariobehling Resolved merge conflicts. Thanks

**Pull request merged**